Corona-Warn-App/src/main/res/layout/vaccination_home_complete_card.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/vaccination_compelete_gradient"
+    android:padding="@dimen/card_padding">
+
+
+    <androidx.appcompat.widget.AppCompatImageView
```

Comment:
`ImageView` is enough  as the one below and let the inflater do the work for you